src/tabwidget.h

```diff
@@ -28,6 +28,7 @@
     #include "dbusaddressable.h"
 #endif
 
+#include "tabbar.h"
```

Comment:
A minor point: as there are only pointers, forward declaration can be used. (i.e., ```class TabBar;```) This reduces a little compiling time.